Update/onik_001.txt

```diff
@@ -4186,6 +4186,7 @@ void main()
 	ClearMessage();
 
 //「レナもシカトすんなー！２人がホテル街へ消えたって回覧す＠
+//「レナもシカトすんなー！２人がラブラブだって回覧す < PS3 voice actually says this＠
```

Comment:
Hmm, if we're going to change the translation to match the PS3 audio, I could change the Japanese text and either change the translation or leave a comment such as
// Update translation!
when I get through onikakushi-hen, I'll apply what there is of the watanagashi-hen patch at the time and play that one through in Japanese.